sdk/storage/azure_storage_blob/tests/blob_client.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core_test::recorded;
+use azure_identity::DefaultAzureCredentialBuilder;
+use azure_storage_blob::{BlobBlobClientGetPropertiesOptions, BlobClient, BlobClientOptions};
+use std::{env, error::Error};
+
+#[recorded::test(live)]
+async fn test_get_blob_properties() -> Result<(), Box<dyn Error>> {
+    // Setup
+    let storage_account_name = env::var("AZURE_STORAGE_ACCOUNT_NAME")
+        .expect("Failed to get environment variable: AZURE_STORAGE_ACCOUNT_NAME");
+    let endpoint = format!("https://{}.blob.core.windows.net/", storage_account_name);
+    let credential = DefaultAzureCredentialBuilder::default().build()?;
```

Comment:
Nit: use `DefaultAzureCredential::new()?`. I didn't even realize we still had the builder in there. `azure_identity` is on my list to clean up for beta.2 and add more credentials.

FWIW - and not a blocker for this PR - but all this stuff will fit nicely with test-proxy. See my `azure_security_keyvault_secrets` tests.